src/scripts/chevron_output_to_file.sh

```diff
@@ -0,0 +1,7 @@
+set -e
+
+# this shell script is very odd because find only lets {} be at the end of the exec command OR only lets it be used once
```

Comment:
Not sure I understand this comment?

Reply:
ohhhhhh...I get it. Can you just indicate in the description that this is a work around for `find -exec`? That's the part I just didn't get 😌 

Might be another way to format all this, but I'm not good enough at Bash to play golf here